.changeset/shaggy-dingos-melt.md

```diff
@@ -0,0 +1,7 @@
+---
+"@neo4j/cypher-builder": patch
+"@neo4j/graphql": patch
```

Comment:
The change doesn't reflect the library at all - should this even have a changesets entry?